src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,38 @@
 import React from 'react';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+
+import { Post } from '../../types/post';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+type Props = {
+  post: Post;
+  posts: Post[];
+};
+
+export const PostInfo: React.FC<Props> = ({ post, posts }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
+      </p>
+    </div>
+
+    <p className="PostInfo__body">
+      {post.body}
+    </p>
+
+    <hr />
+
+    {post.comment ? (
+      <CommentList posts={posts} />
```

Comment:
Hm, I think some mistake here, your `<CommentList />` accept `posts` but as I understand it should receive `comment`, no? 

Due to this you can compare posts on your page and see same content in comment blocks.